experimental/deploy_cycle_slurm_ndv4/prereqs.json

```diff
@@ -71,6 +75,24 @@
         "variables.mariadb_password_secret_name"
       ]
     },
+    {
+      "type": "local_script",
+      "script": "add_secret_in_keyvault.sh",
+      "args": [
+        "variables.key_vault",
+        "variables.log_analytics_ws_customer_id_secret_name",
+        "variables.log_analytics_ws_customer_id_secret_value"
```

Comment:
It would be safer to have the `add_secret_in_keyvault.sh` script to generate a random secret instead of passing one as parameter.

Reply:
This has to be a specific log analytics ws key and customer_id (not a random one)